src/test/java/apoc/couchbase/CouchbaseIT.java

```diff
@@ -0,0 +1,210 @@
+package apoc.couchbase;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+import static org.junit.Assume.assumeTrue;
+
+import java.util.List;
+import java.util.Map;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.neo4j.graphdb.GraphDatabaseService;
+import org.neo4j.graphdb.QueryExecutionException;
+import org.neo4j.test.TestGraphDatabaseFactory;
+
+import com.couchbase.client.java.document.JsonDocument;
+import com.couchbase.client.java.document.json.JsonObject;
+
+import apoc.util.TestUtil;
+
+public class CouchbaseIT extends CouchbaseAbstractTest {
+
+  protected static GraphDatabaseService graphDB;
+
+  @BeforeClass
+  public static void setUp() throws Exception {
+    CouchbaseAbstractTest.setUp();
+    if (couchbaseRunning) {
+      graphDB = new TestGraphDatabaseFactory().newImpermanentDatabaseBuilder().newGraphDatabase();
+      TestUtil.registerProcedure(graphDB, Couchbase.class);
+    }
+  }
+
+  @AfterClass
+  public static void tearDown() {
+    CouchbaseAbstractTest.tearDown();
+    if (graphDB != null) {
+      graphDB.shutdown();
+    }
+  }
+
+  @Test
+  @SuppressWarnings("unchecked")
+  public void testGetViaCall() throws Exception {
+    assumeTrue(couchbaseRunning);
```

Comment:
Just tested this way (+ removed from every @test method) and it works fine:
```  @BeforeClass
  public static void setUp() throws Exception {
    CouchbaseAbstractTest.setUp();
    assumeTrue(couchbaseRunning);
  }
```
If you like I can commit this way.